app/src/main/java/com/app/worte/worte/FileSystemOperator.java

```diff
@@ -37,14 +44,16 @@ private void parseDbFile(File dbFile)
             String line = reader.readLine();
             while (line != null)
             {
-                String[] splited = line.split(" - ");
+                String[] splited = line.split(" \\| ");
 
-                if(splited.length == 2)
+                if(splited.length == 3)
```

Comment:
Magic number.
Consider making it a constant with an appropriate comment.

Reply:
Fixed